test/coverlet.integration.tests/DotnetTool.cs

```diff
@@ -29,5 +29,20 @@ public void DotnetTool()
             Assert.Contains("Test Run Successful.", standardOutput);
             AssertCoverage(clonedTemplateProject, standardOutput: standardOutput);
         }
+
+        [ConditionalFact]
+        [SkipOnOS(OS.Linux)]
+        [SkipOnOS(OS.MacOS)]
```

Comment:
Curious. Why do we only support Windows?

Reply:
we've an open issue for that, never had time to investigate https://github.com/coverlet-coverage/coverlet/issues/660